Makefile

```diff
@@ -1,5 +1,5 @@
 SHELL=/bin/bash
-PYTHON=python3.10
+PYTHON=python3.11
```

Comment:
Should this be part of this PR?

Reply:
Not necessarily, but it isn't a problem either because it is a separate patch. While implementing this, I ran into some issues and wanted to use the new exception pretty printing of Python 3.11.